README.md

```diff
@@ -140,7 +140,8 @@ the `LovelyEditor` with a new valid editorState. You can find a similar example
 [in our Storybook](https://lovely-editor.netlify.com/?selectedKind=App%20Example&selectedStory=with%20an%20example%20Menu%20and%20EditorQuill%20Block&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel).
 
 ```js
-import { LovelyEditor, EditorQuill, EditorTui } from 'lovely-editor'
+import { LovelyEditor } from 'lovely-editor'
+import { EditorQuill } from 'lovely-editor/dist/components/editor-quill'
```

Comment:
In the Readme of Stefan's PR it is stated to be imported this way:
```
import { EditorQuill } from 'lovely-editor/components/editor-quill'
```
Would be great if this works too.

https://github.com/lovelysystems/lovely-editor/pull/54/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R31

Reply:
I just checked out the Stefan's branch and run `npm run build`, then tried to import `EditorQuill` - it was also only possible by specifying `/dist/` (or `/src/`) in the path.